src/network/serverpackethandler.cpp

```diff
@@ -471,6 +471,12 @@ void Server::process_PlayerPos(RemotePlayer *player, PlayerSAO *playersao,
 	v3f position((f32)ps.X / 100.0f, (f32)ps.Y / 100.0f, (f32)ps.Z / 100.0f);
 	v3f speed((f32)ss.X / 100.0f, (f32)ss.Y / 100.0f, (f32)ss.Z / 100.0f);
 
+	const v3s16 old_node_position = floatToInt(playersao->getBasePosition(), BS);
+	const v3s16 new_node_position = floatToInt(position, BS);
```

Comment:
Please can these 2 variables have shorter names for easier coding?
`new_nodepos` `old_nodepos` is ok.

Reply:
Sure, sounds okay